api/src/user/controllers/user.controller.ts

```diff
@@ -248,7 +248,9 @@ export class ReadWriteUserController extends ReadOnlyUserController {
         roles: (await this.roleService.findAll())
           .filter((role) => user.roles.includes(role.id))
           .map((role) => role.id),
-        avatar: (await this.attachmentService.findOne(user.avatar))?.id,
+        avatar: user.avatar
+          ? (await this.attachmentService.findOne(user.avatar))?.id
+          : undefined,
```

Comment:
avatar should be either ObjectId or null

Reply:
I have update the logic to support null value